src/views/Course.vue

```diff
@@ -147,12 +147,16 @@
                 </v-col>
               </v-row>
               <v-row v-if="showSignUpForm && !$currentUser.isAdmin" no-gutters align="start">
-                <v-col md="auto">
-                  <v-checkbox id="agree-to-terms-checkbox" v-model="agreedToTerms" class="mt-0"></v-checkbox>
-                </v-col>
-                <v-col>
+                <v-col order="12">
                   <TermsAgreementText class="pt-1" />
                 </v-col>
+                <v-col md="auto">
+                  <v-checkbox
+                    id="agree-to-terms-checkbox"
+                    aria-labelledby="permission-agreement"
+                    v-model="agreedToTerms"
+                    class="mt-0"></v-checkbox>
+                </v-col>
```

Comment:
Added fix for https://jira.ets.berkeley.edu/jira/browse/DIABLO-538. Order of the two `v-col`  above has been reversed but UI is same.